### ISSUE TITLE
support csv with no header?

### ISSUE THREAD

**Issue by AlJohri** [lzakharov/csv2md#11]
[csvkit](https://csvkit.readthedocs.io/en/latest/) supports a [common cli flag](https://csvkit.readthedocs.io/en/latest/common_arguments.html) across all of its tools to support inputting csv files with no header

```
-H, --no-header-row   Specify that the input CSV file has no header row.
                      Will create default headers (a,b,c,...).
```

it would be helpful if `csv2md` supported the same or a similar flag

this would allow quickly creating csv's using jq's `@csv` filter and piping it into `csv2md`

**Comment by lzakharov** [comment]
Good idea! Would you like to send PR with this feature? If no, I'll try to implement it on the next week.

**Comment by lzakharov** [comment]
Implemented in `v1.2.0`. Thanks for the suggested feature!
